.changeset/perfect-doors-switch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/icons': major
+---
+
+run changeset to update icons
```

Comment:
The changeset should describe the change a commit or pull request introduces. It will be added to the changelog to inform users what changed in a release and how they might have to adapt their code. Read more in the [changeset docs](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md).
```suggestion
Added new Employee, Employees and EmployeeSideBySide icons in size 16.
```